### PR TITLE
feat: add `chat_info` bot method

### PR DESCRIPTION
**Pull request by YunusovSamat**
# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .coverage
 .env
 .venv/
+venv/
 __pycache__
 htmlcov
 site
+.idea/
+
```
Comment (lxmnk) [inline .gitignore:9]: Empty line.

File: botx/client/chats_api/chat_info.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    Attributes:
+        is_admin: Is user admin.
+        server_id: User server id.
+        huid: User huid.
+        kind: User type.
+    """
+
+    is_admin: bool
+    server_id: UUID
```
Comment (lxmnk) [inline botx/client/chats_api/chat_info.py:39]: This field does not provide useful information. Remove it, pls.

File: botx/client/chats_api/chat_info.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        inserted_at: Chat creation datetime.
+        members: Chat members.
+        name: Chat name.
+    """
+
+    chat_type: ChatTypes
+    creator: UUID
+    description: Optional[str]
+    chat_id: UUID
+    inserted_at: dt
```
Comment (lxmnk) [inline botx/client/chats_api/chat_info.py:62]: `created_at`

File: botx/client/chats_api/chat_info.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        creator: Chat creator.
+        description: Chat description.
+        chat_id: Chat id.
+        inserted_at: Chat creation datetime.
+        members: Chat members.
+        name: Chat name.
+    """
+
+    chat_type: ChatTypes
+    creator: UUID
```
Comment (lxmnk) [inline botx/client/chats_api/chat_info.py:59]: `creator_id`

File: botx/client/chats_api/chat_info.py [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    @classmethod
+    def from_domain(cls, chat_id: UUID) -> "BotXAPIChatInfoRequestPayload":
+        return cls(group_chat_id=chat_id)
+
+
+class BotXAPIChatInfoMember(VerifiedPayloadBaseModel):
+    admin: bool
+    server_id: UUID
+    user_huid: UUID
+    user_kind: BotAPIUserKinds
```
Comment (lxmnk) [inline botx/client/chats_api/chat_info.py:79]: Pls move this enum to shared models.

**Review by lxmnk — CHANGES_REQUESTED** [review]

File: botx/client/chats_api/chat_info.py [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+    group_chat_id: UUID
+
+    @classmethod
+    def from_domain(cls, chat_id: UUID) -> "BotXAPIChatInfoRequestPayload":
+        return cls(group_chat_id=chat_id)
+
+
+class BotXAPIChatInfoMember(VerifiedPayloadBaseModel):
+    admin: bool
+    server_id: UUID
```
Comment (lxmnk) [inline botx/client/chats_api/chat_info.py:77]: Remove.

**Review by lxmnk — APPROVED** [review]

**Review by Nimond — APPROVED** [review]

**Pull request merged**